hera_cal/apply_cal.py

```diff
@@ -74,7 +74,8 @@ def recalibrate_in_place(data, data_flags, new_gains, cal_flags, old_gains=None,
 
 
 def apply_cal(data_infilename, data_outfilename, new_calibration, old_calibration=None, flags_npz=None, 
-              filetype='miriad', gain_convention='divide',  add_to_history='', clobber=False, **kwargs):
+              flag_nchan_low = 50, flag_nchan_high = 50, filetype='miriad', gain_convention='divide',  
```

Comment:
I think the default in the module should be to not flag band edges.

Reply:
We want to set the default for now to 50 so that we can rerun part of the data without changing the makeflow. I have a separate PR to change the makeflow for future runs that will parameterize this there. If you want to open an issue to change it in the future, I'm OK with that, but I want it for the next week or so like this.

Reply:
Can you rerun on a branch? It seems really odd to put temporary defaults into the master branch for a specific run.